__tests__/app.test.js

```diff
@@ -96,70 +96,121 @@ describe("/api/topics", () => {
   });
 });
 
-describe("/api/articles", () => {
+describe.only("/api/articles", () => {
```

Comment:
Please get rid of any .onlys before running tests

Reply:
Sorry!